tests/common/find_service_setup.rs

```diff
@@ -39,6 +39,10 @@ impl FindService {
         let bind = "127.0.0.1:8080";
 
         let _proc = tokio::process::Command::new(path)
+            .arg("-s")
+            .arg("5")
+            .arg("-t")
+            .arg("5")
```

Comment:
Wasn't sure if this was a reasonable global setting, but we likely want to make this configurable on a per-test basis in the future.